### ISSUE TITLE
v4.9.3.5 upgrade causes UI display problems

### ISSUE THREAD

**Issue by yoyofreeman** [RainbowMiner/RainbowMiner#2842]
I use server/client mode. This upgrade causes the server-side webui to be unable to display the client list and information, and the client webui is unable to display the list of other machines in the same group. Only the local miner can be displayed.

The client list can be seen in the cli on the server side, but this list is useless to me.

Please help me solve this problem. Thank you.

**Comment by whitindllc** [comment]
yeah looks like both the dashboard in the gui disappeared and rbminer/net/monitoring is gone

**Comment by whitindllc** [comment]
yeah i kept a version from upgrading... the section for remote workers is gone in the dashboard section

**Comment by RainbowMiner** [comment]
This was related to issue #2843

**Comment by whitindllc** [comment]
yes dashboard client list is back in gui

**Comment by yoyofreeman** [comment]
> [这与问题#2843](https://github.com/RainbowMiner/RainbowMiner/issues/2843)相关[](https://github.com/RainbowMiner/RainbowMiner/issues/2843)

Thank you very much, the problem has been solved and the client list is displayed normally.

**Comment by maiquecoutinho** [comment]
new workers not show on monitor.

**Comment by RainbowMiner** [comment]
See issue #2847 - is everything back to normal now?

**Comment by RainbowMiner** [comment]
This was against v4.9.3.5. The web UI was rewritten on 2026-07-15 (d4c520a7) and now uses Tabulator 6.3.1 instead of the old table code. Closing as obsolete for the current master (v5.0.2.1) - please open a new issue if this still occurs. Thanks for the report!
